lib/PayPal/Api/Payment.php

```diff
@@ -544,7 +544,19 @@ public function getApprovalLink()
     {
         return $this->getLink(PayPalConstants::APPROVAL_URL);
     }
-
+	
+	/**
+     * Get token from Approval Link
+     *
+     * @return null|string
+     */
+	public function getToken()
+	{
+		$parameter_name = "token";
+		parse_str(parse_url($this->getApprovalLink(), PHP_URL_QUERY), $query);
+		return (!isset($query[$parameter_name]) || is_null($query[$parameter_name])) ? null : $query[$parameter_name];
```

Comment:
Based on PHP docs: `isset` determines if a variable is set and is not NULL. We do not need additional is_null check here.